### DISCUSSION TITLE
Python - Inference script

### DISCUSSION THREAD

**Discussion by Natriumpikant** [ultralytics/ultralytics#969] [Q&A] [answered]
Hello together,

For a project at university i need a script, which creates in python inference results and prints them in console (not in a file). 

I could read out the created result-file but i think this is not the best way.

I worked with Darknet Yolo before and a code like this to create inferencing and printing the results: 

Thanks for help :)

**Code in Darknet:**
https://pastebin.com/GX0cMQe1

**Comment by Natriumpikant** [comment, marked as answer]
My fault - can ease be achieved with this instructions.
https://docs.ultralytics.com/python/

**Reply by glenn-jocher** [reply]
@Natriumpikant hello!

Absolutely, you can achieve inference with YOLOv8 using Python and print the results directly to the console. The instructions you've found in the Ultralytics Docs are the right place to start. 

To summarize, you'll need to load the YOLOv8 model using the appropriate Python commands, and then you can pass your images or videos to the model for inference. The results can be accessed from the returned object and printed to the console. This way, you can display the inference results in real-time without writing them to a file.

If you have any further questions or need assistance with specific parts of the process, feel free to ask. Good luck with your university project! 🎓🚀
